src/Grids/rectilinear_grid.jl

```diff
@@ -423,37 +423,21 @@ return_metrics(::RectilinearGrid) = (:xᶠᵃᵃ, :xᶜᵃᵃ, :yᵃᶠᵃ, :y
 
 
 #####
-##### Get minima of grid
+##### Grid spacings
 #####
 
-function min_Δx(grid::RectilinearGrid)
-    topo = topology(grid)
-    if topo[1] == Flat
-        return Inf
-    else
-        return min_number_or_array(grid.Δxᶜᵃᵃ)
-    end
-end
+Δx(::Type{Center}, grid::RectilinearGrid) = topology(grid)[1] == Flat ? Inf : grid.Δxᶜᵃᵃ
```

Comment:
I don't think we want to use these names, and we also don't want to define "one-location" functions. This was a mistake with the xnodes, ynodes, etc functions.